UnoCpp/UnoCpp/Player.h

```diff
@@ -1,14 +1,13 @@
 #pragma once
-#include <memory>
 #include <string>
 
-class CardCollection;
+#include "CardCollection.h"
```

Comment:
alguma razão pra ser include e não forward declaration?

Reply:
Porque agora eu não to guardando mais como ponteiro, ai precisa do include